BeagleBone_black/src/sensors/mpu9250.hpp

```diff
@@ -39,10 +39,15 @@ using hyped::utils::io::I2C;
 
 namespace sensors {
 
-class MPU9250 : ImuInterface {
+class MPU9250 : public ImuInterface {
  public:
   MPU9250(Logger& log, uint32_t pin, bool isSpi, uint8_t i2c_addr);
   ~MPU9250();
+
+  bool isOnline() override {
+    return whoAmI();
+  }
```

Comment:
These definitions seem to be a bit inconsistent across files. Usually, it's all on one line, here it's 3 lines. Also if the definition doesn't look good on a single line in class declaration, I would move it to the `.cpp` file but I don't think the style guide is strict on that so I'll leave it up to you.

Reply:
thanks, I actually expect this one to be moved to .cpp at some point. I would need to consult Jack to figure out what is a good way to determine if a mpu sensor is online.